File: Poin A/RAM.cpp
#include<iostream>

using namespace std;

class RAM{
    private :
        //atribut
        string capacity;
        int priceR;
    public :
        //konstruktor
        RAM(){}
        RAM(string c, int p){
            this->capacity = c;
            this->priceR = p;
            
        }

        //setter & getter
        void setCapacity(string c){
            this->capacity = c;
        }
        void setPriceR(int p){
            this->priceR = p;
        }


        string getCapacity(){
            return this->capacity;
        }

        int getPriceR(){
            return this->priceR;
        }


        //output
        void printRAM(){
            cout << "RAM          : " << this->getCapacity() << endl;
            cout << "Harga        : " << this->getPriceR() << endl;


        }

        ~RAM(){}

};

File: Poin B.php/RAM.php
<?php
class RAM
{
    //atribute
    private $capacity;
    private $price;

    //construktor
    function __construct($capacity, $price)
    {
        $this->capacity = $capacity;
        $this->price = $price;
    }

    //get set methods
    function setCapacity($capacity)
    {
        $this->capacity = $capacity;
    }

    function  getCapacity()
    {
        return $this->capacity;
    }

    function setPrice($price)
    {
        $this->price = $price;
    }

    function getPrice()
    {
        return $this->price;
    }

    // output method
    function output()
    {   
        echo "RAM                       : ". $this->getCapacity() . "<br>";
        echo "Harga                     : ". $this->getPrice() . "<br>";
    }

    //destructor
    function __destruct()
    {
    }
}